rc/filetype/just.kak

```diff
@@ -38,18 +38,22 @@ define-command -hidden just-indent-on-new-line %{
 # ‾‾‾‾‾‾‾‾‾‾‾‾
 
 add-highlighter shared/justfile regions
+
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/content/recipe regex '^@?([\w-]+)([^\n]*):(?!=)([^\n]*)' 1:function 2:meta 3:keyword
+add-highlighter shared/justfile/content/assignments regex ^([\w-]+\h*:=\h*[^\n]*) 1:meta
+add-highlighter shared/justfile/content/operator regex '((^@|:=|=|\+|\(|\)))' 1:operator
+add-highlighter shared/justfile/content/strings regions
+add-highlighter shared/justfile/content/strings/dq region '"' (?<!\\)(\\\\)*" fill string
```

Comment:
Is there an example where this behaves differently from the old `shared/justfile/double_string`?
If yes, it should probably still be `shared/justfile/content/doule_string` to match other `rc` files.

Reply:
Yes, the golabl spaced region indiscriminately brakes the content group.

so this hfailed - which was the motivation to get this PR started:

`ssh m="":` - no syntax highlighting was done because the `""` string region broke up the possiblilty do match the line in the default region.

Reply:
> If yes, it should probably still be shared/justfile/content/doule_string to match other rc files.

the region needs its own name. it would be 

``shared/justfile/content/strings/doule_string` &mdash; is that what you suggest for more homogenous grepping of the codebase?

Reply:
Oh yeah, that makes sense!

Reply:
> the region needs its own name. it would be

I see, maybe `strings/double` but it's also fine to leave it like this imo

Reply:
ok adopted: 
- `strings/double`
- `strings/single`